04-onion-routing.md

```diff
@@ -189,6 +189,9 @@ This is formatted according to the Type-Length-Value format defined in [BOLT #1]
 
 1. `tlv_stream`: `payload`
 2. types:
+    1. type: 1 (`invoice_request`)
```

Comment:
Does this contain the paths that an LSP uses to send the `held_htlc_available` onion message to?